conf/app.ini

```diff
@@ -453,6 +455,8 @@ SCHEDULE = @every 24h
 ; Create new users, update existing user data and disable users that are not in external source anymore (default)
 ;   or only create new users if UPDATE_EXISTING is set to false
 UPDATE_EXISTING = true
+; Replace all Public SSH Keys that has been added manually with keys from LDAP (default false)
+REPLACE_PUBLIC_SSH_KEYS = false
```

Comment:
I agree. Either such option and/or:
If there's a need to delete the keys that users have already added manually, it seems better to create some sort of button for admins. Such as "delete all keys synchronized from source x for [user-y|all-users]".

But this can also be performed directly in the database for now, I guess the need for this wouldn't be recurring and I'm not sure that it's needed at all.

I'll remove this and create a new PR for such option if I see that there's any need to disable/remove keys